### DISCUSSION TITLE
Unable to get the Julia pras code to work

### DISCUSSION THREAD

**Discussion by wshobe** [NREL/ReEDS-2.0#198] [Q&A: Execution]
My model is consistently failing at the call to the Julia code. This problem crops up in any case I run, including the cases_test.csv Pacific and ERCOT runs. I am running on a Linux HPC system (with slurm). The ReEDS-2.0 project folder has Julia set up with all the required packages.  This error happens even if I set pras=0. I'm stuck and can't get any results. Any help/advice would be greatly appreciated.

[slurm-848182.txt](https://github.com/user-attachments/files/18155208/slurm-848182.txt)

**Comment by kennedy-mindermann** [comment]
It looks like some julia packages are not being found. You could try walking through the [julia setup instructions](https://nrel.github.io/ReEDS-2.0/setup.html#reeds2pras-julia-and-stress-periods-setup) again. Otherwise, if you would like to run ReEDS without PRAS, you'll want to set `GSw_PRM_CapCredit = 1`. You shouldn't need to change any other switch.

**Reply by wshobe** [reply]
I am getting the same failure even with GSw_PRM_CapCredit = 1. I have uploaded my cases file and gamslog.txt. 
[cases_pjm_historic.csv](https://github.com/user-attachments/files/18156772/cases_pjm_historic.csv)
[gamslog.txt](https://github.com/user-attachments/files/18156820/gamslog.txt)

Also, I have been through the Julia setup a couple of times. Do you have a diagnostic test I might do to check the setup?

**Reply by wshobe** [reply]
It does not appear to be sufficient to set GSw_PRM_CapCredit to 1. In Augur.py, the logical test in  
 if pras_this_solve_year or (not int(sw.GSw_PRM_CapCredit)):
 evaluates to true if pras is 2 or under other circumstances, notwithstanding the value of the GS switch. So, since the default is pras=2, you need to set pras=0 as well. I did that and ran without PRAS. Now I'll try to figure out why my PRAS function isn't working.
